components/service/glean/metrics.yaml

```diff
@@ -99,14 +99,18 @@ baseline:
   #   notification_emails:
   #     - telemetry-client-dev@mozilla.com
 
-  # accessibility_services:
-  #   type: string_list
-  #   description: |
-  #     List of accessibility services enabled on the device.
-  #   bugs:
-  #     - 1497894
-  #   notification_emails:
-  #     - telemetry-client-dev@mozilla.com
+  a11y_services:
```

Comment:
We also need to rename this field in `baseline.md`, where's it's written as `accessibility_services`